tests/e2e/lib/blocks/simple-payments.js

```diff
@@ -30,17 +29,17 @@ export default class SimplePaymentBlock {
 		const priceSelector = this.getSelector( '.simple-payments__field-price' );
 		const emailSelector = this.getSelector( '.simple-payments__field-email' );
 
-		await waitAndClick( this.page, titleSelector );
-		await waitAndType( this.page, titleSelector, title );
+		await page.click( titleSelector );
```

Comment:
Are we sure that `page` would always be the same as `this.page`?

Reply:
It will use the global `page`. Still, I will check all files and refer to `this.page` as it should be.